### PR TITLE
Basic Login System

### PR DESCRIPTION
**Pull request by williamschen23**

### CODE REVIEW

**Comment by williamschen23** [comment]
closes #4

File: src/app/api/auth/[...nextauth]/options.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            name: "Credentials",
+            credentials: {
+                username: {
+                    label: "Username:",
+                    type: "text",
+                    placeholder: "dev"
+                },
+                password: {
+                    label: "Password:",
+                    type: "text",
```
Comment (RichardJSun) [inline src/app/api/auth/[...nextauth]/options.ts:16]: Should be resolved by #21 but the type of password should be password
```suggestion
                    type: "password",
```

**Review by RichardJSun — APPROVED** [review]

File: src/app/api/auth/[...nextauth]/options.ts (hunk 1 above)
Comment (williamschen23) [reply src/app/api/auth/[...nextauth]/options.ts:16]: oops

Comment (williamschen23) [reply src/app/api/auth/[...nextauth]/options.ts:16]: shouldnt matter for now

**Pull request merged**